content/docs/operating/integrations.md

```diff
@@ -55,6 +55,7 @@ data volumes.
   * [QuasarDB](https://doc.quasardb.net/master/user-guide/integration/prometheus.html): read and write
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
   * [Splunk](https://github.com/kebe7jun/ropee): read and write
+  * [Sysdig](https://sysdig.com/opensource/prometheus): read
```

Comment:
To be clear a normal Prometheus can read data from Sysdig via the remote read protocol?

We've not had one of those before for this list, and I'm not seeing any relevant docs on the linked page so I'm just double checking.

Reply:
Sysdig does not support Prometheus Remote Read API, but it exposes Prometheus HTTP API endpoint for querying the data.

Reply:
I've also updated the link

Reply:
This list is for remote storage integrations, we don't have a list for things that provide the HTTP API I'm afraid.